extension/source/QuillPage/index.tsx

```diff
@@ -4,5 +4,8 @@ import '../ContentScript';
 import '../styles/index.scss';
 import './styles.scss';
 import QuillPage from './QuillPage';
+import Browser from 'webextension-polyfill';
+
+(window as any).Browser ??= Browser;
```

Comment:
Add this to `window.d.ts`.

```suggestion
window.Browser ??= Browser;
```

Reply:
Will do.

Reply:
Done.